focus-blocked/js/init.js

```diff
@@ -1,3 +1,12 @@
+/************** sentry **********************/ // function isValidUrl(string) {
```

Comment:
Why is this commented out?

Reply:
This was a test; the primary implementation is located in `utils.js`.